scripts/knot_benchmark/src/benchmark/cases.py

```diff
@@ -204,6 +204,8 @@ def install(self, dependencies: list[str]):
             "--python",
             self.python,
             "--quiet",
+            "--exclude-newer",
+            "2024-09-03T00:00:00Z",
```

Comment:
Should we make this a more prominent constant that's easier to find / is defined in `projects`?

Reply:
I think in practice it would only make it harder to discover the value of the constant if we defined it as a global somewhere -- if I wanted to find out the exact command passed when installing dependencies I would grep for `"uv pip"` or `"pip"` in the source code, and I'd find this list of arguments fairly easily. (I know this because it's exactly what I did while debugging https://github.com/astral-sh/ruff/pull/13228 ;) But if this argument was defined in another file, I'd then have to add an additional step of going to look in that other file.

I'll add a comment about why we pass the argument, however!

Reply:
I would find a comment at the top of projects useful just in case someone wonders why the dependencies are all outdated